frontend/components/organisms/CourseDetailAccordionMenu.tsx

```diff
@@ -78,7 +78,7 @@ export function CourseDetailAccordionMenu({
           courseData.sections.map((section, sectionIndex) => {
             const sectionProgress = getSectionCompletionPercentage(
               sectionIndex,
-              session.user.id,
+              session?.user?.id,
```

Comment:
```suggestion
              session.user.id,
```

これでエラーになります？
ならなかったらこれで行きたいです。
下の変更箇所も同様に

Reply:
@issiyda 
レビューありがとうございます。
オプショナルチェーンないとエラーでてしまいます。。。。